src/applications/personalization/dashboard-2/sass/dashboard.scss

```diff
@@ -5,7 +5,20 @@
 }
 
 .dashboard {
-  .health-care {
+  .health-care-wrapper {
+
+    .health-care {
+      width: 520px;
```

Comment:
I am not a fan of setting a fixed width, however the designs appear to call for it. The width of the section seems to remain equal on small Desktop and tablet.

Reply:
I have a feeling the mocks are not going to be the source of truth for the width. We're probably going to want to revisit this and go back to making the widths flexible. Would also be nice to code this in a mobile-first manner (ie, don't use `max-width`). So, `width: 100%;` by default and then use a `min-width: $small-screen` to set the width to `520px` and keep the `min-width: $large-screen` that you have.

Reply:
Alright, I made the update and triple checked that the layout still looks the same 👍